src/alto-ui/Datagrid/components/DatagridCell/DatagridCell.scss

```diff
@@ -107,6 +106,16 @@
 
   @extend %text-ellipsis;
 
+  &--header {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+  }
+
+  &:not(.DatagridCell__content--header) {
```

Comment:
Please inverse the logic, don't remove line height from main class, just apply proper value for `.DatagridCell__content--header`

Reply:
thanks, it's done